literature-cleaned.bib

```diff
@@ -13234,6 +13314,15 @@ @article{BSR04
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2003 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 
+@book{F03,
+  title = {{UML Distilled: A Brief Guide to the Standard Object Modeling Language (3rd Edition)}},
+  author = {Fowler, Martin},
+  edition = 3,
+  publisher = {Addison-Wesley Professional},

```

Comment:
Can we reuse `AddisonWesleyLongman` variable here or is it a bad match?

Reply:
I was not sure on that. After further research, I think it using the variable should be correct.

Reply:
Ok, proceed as you like. :)